app/Domain/Integrations/Mappers/OrganizerMapper.php

```diff
@@ -13,7 +14,7 @@ final class OrganizerMapper
     /**
      * @return Organizer[]
      */
-    public static function map(RequestActivationRequest $request, string $id): array
+    public static function map(RequestActivationRequest|UpdateIntegrationOrganizersRequest $request, string $id): array
```

Comment:
On other Mappers we added a mapUpdate for example and it passes it to map. Then you can only pass 1 type.
Either approach is fine to me